getting-started-publishing/publishing-intro.md

```diff
@@ -0,0 +1,45 @@
+---
+info_no: Side om "Komme i gang - Publisere beskrivelser/kataloger"
+description: Skal gi overordnet informasjon om hvordan publisere beskrivelser/kataloger til data.norge.no
+url: https://data.norge.no/getting-started/publishing-data
+note: Vurder å slå sammen med "Creating descriptions"
+url: https://data.norge.no/getting-started/about-publishing
+---
+
+# Lage og publisere beskrivelser
+
+I denne guiden vil du lære
+
+- hvordan lage beskrivelser,
+- hvordan publisere beskrivelsene til data.norge.no, og
+- om hvilke tilganger du trenger.
+
+Det er to måter du kan lage og publisere databeskrivelser:
+
+1. Du lager beskrivelsen i korrekt format selv og registrerer et høsteendepunkt data.norge.no kan høste beskrivelsen fra.
+2. Du bruker data.norge.no sin registreringsløsning for å registrere beskrivelser.
+
+## 1. Høste fra eget endepunkt
+
+Om du ønsker å forvalte beskrivelsene selv, må du lage dem og tilgjengeliggjøre dem.
+
+[Hvordan lage beskrivelser]().
+
+[Hvordan registrere endepunkt for høsting]().
+
+## 2. Registreringsløsning på data.norge.no
+
+Data.norge.no tilbyr et grafisk grensesnitt for å lage beskrivelser av datasett, datatjenester, begrep og tjenester. Informasjonsmodeller og hendelser er foreløpig ikke støttet.
+Dette er en skjemaløsning som lar deg fylle ut relevante felt og som genererer beskrivelser i korrekt format som deretter automatisk høstes og publiseres til data.norge.no.
+
+[Hvordan lage beskrivelser med registreringsløsningen]()
+
+## Tilganger og bruksvilkår
```

Comment:
Duplikat informasjon. Eventuelt lenke til side om innlogging/tilganger.

Reply:
Ja, endte opp med det to steder. Ideelt sett funker intro-siden som en sjekkliste som gir en umiddelbart oversikt over alle stegene som skal til for å komme i mål. Kan ha det kun der, ev.